frontend/src/components/Scratch/hooks/default-clang-format.yaml

```diff
@@ -0,0 +1,52 @@
+# Based on https://github.com/pmret/papermario/blob/main/.clang-format
+
+DisableFormat: true # clang-format supports many filetypes
+---
+Language: Cpp
+DisableFormat: false
+BasedOnStyle: WebKit
+IndentWidth: 4
+UseTab: Never
+ColumnLimit: 120
+
+AllowShortIfStatementsOnASingleLine: false
+AllowShortBlocksOnASingleLine: false
+
+BreakBeforeBraces: Custom
+BraceWrapping:
+  # Place opening brace on next line if the statement is multi-line
+  # e.g.
+  #  if (very
+  #      long
+  #      condition)
+  #  {
+  AfterControlStatement: MultiLine
+
+IndentCaseLabels: true
+
+BreakBeforeBinaryOperators: NonAssignment
+
+ExperimentalAutoDetectBinPacking: true
+BinPackParameters: false
+AlignAfterOpenBracket: BlockIndent
+
+AlignConsecutiveAssignments:
+  Enabled: true
+  AcrossEmptyLines: false
+  AcrossComments: false
+  AlignCompound: true
+  PadOperators: true
+AlignConsecutiveDeclarations: # structs etc
+  Enabled: true
+  AcrossEmptyLines: false
+  AcrossComments: false
+AlignConsecutiveMacros:
+  Enabled: true
+  AcrossEmptyLines: false
+  AcrossComments: false
+AlignTrailingComments: true
```

Comment:
I think some / all of these are the ones I'd prefer to just remove (assuming the default is to not align)

Reply:
you want `AlignConsecutiveAssignments` off I think

Reply:
will need to carry these over to papermario too; discuss with clover